samples/ask-akka-agent/README.md

```diff
@@ -2,36 +2,33 @@
 
 This sample illustrates how to build an AI agent that performs a RAG workflow. 
 
-# OpenAi and MongoDb Atlas
+## Running the app
 
 This sample requires OpenAI API Key and a MongoDb Atlas URI. 
 
-## Mongo Atlas
+#### Mongo Atlas
```

Comment:
Isn't this the third level of header, so should be `###` since "Running the app" is `##`